release-notes/opendistro-for-elasticsearch-performance-analyzer.release-notes-1.13.0.0.md

```diff
@@ -0,0 +1,25 @@
+## 2020-11-17 Version 1.13.0.0 (Current)
```

Comment:
Might need to fix the date. Also, can we add rca commits if any to the same release notes?